website/views.py

```diff
@@ -4399,3 +4403,87 @@ def deletions(request):
         "monitor.html",
         {"form": form, "deletions": Monitor.objects.filter(user=request.user)},
     )
+
+
+def generate_bid_image(request, bid_amount):
+    image = Image.new("RGB", (100, 50), color="white")
+    draw = ImageDraw.Draw(image)
+
+    font = ImageFont.load_default()
+    draw.text((10, 10), f"Bid: ${bid_amount}", fill="black", font=font)
+
+    byte_io = io.BytesIO()
+    image.save(byte_io, format="PNG")
+    byte_io.seek(0)
+
+    return HttpResponse(byte_io, content_type="image/png")
+
+
+def SaveBiddingData(request):
+    form = BidForm()
+    if request.method == "POST":
+        url = request.POST.get("issue_url")
+        amount = request.POST.get("bid_amount")
+        user = request.POST.get("user")
+        current_time = datetime.now(timezone.utc)
+        bid = Bid(
+            issue_url=url,
+            amount=amount,
+            created_at=current_time,
+            modified_at=current_time,
+            user=user,
+        )
+        bid.save()
+        return render(request, "bidding.html")
+
+    return render(request, "bidding.html")
+
+
+def fetch_current_bid(request):
+    if request.method == "POST":
+        data = json.loads(request.body)
+        issue_url = data.get("issue_url")
+        bid = Bid.objects.filter(issue_url=issue_url).order_by("-created_at").first()
+        if bid is not None:
+            bid_amount = bid.amount
+            time_left = bid.created_at - datetime.now(timezone.utc)
+            user = bid.user
+            date = bid.created_at
+            status = bid.status
+            return JsonResponse(
+                {
+                    "current_bid": bid_amount,
+                    "time_left": (time_left.total_seconds() + 86400),
+                    "date": date,
+                    "user": user,
+                    "status": status,
+                }
+            )
+        else:
+            return JsonResponse({"error": "Bid not found"}, status=404)
+    else:
+        return JsonResponse({"error": "Method not allowed"}, status=405)
+
+
+def submit_pr(request):
+    form = BidForm()
```

Comment:
Same here -- your form is unused and you create `Bid` instance directly from the POST request data.